lib/castle/podcast.ex

```diff
@@ -20,15 +20,12 @@ defmodule Castle.Podcast do
     |> cast(attrs, [:account_id, :title, :subtitle, :image_url, :created_at, :updated_at, :published_at])
   end
 
-  def recent(accounts, limit, page) when is_list(accounts) do
-    offset = (page - 1) * limit
-    Castle.Repo.all(from p in Castle.Podcast, where: p.account_id in ^accounts,
-      limit: ^limit, offset: ^offset, order_by: [asc: :title])
```

Comment:
Seems like this `order_by` may have been lost?

Reply:
Nice catch! Yes, this was blown away when refactoring the pagination.